### PR TITLE
Tickets/dm 22528

### PR DESCRIPTION
**Pull request by leannep**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -356,6 +356,16 @@ Datasets considered but not selected
...
 
+===========================================
+Timescale for preserving processed datasets
+===========================================
+
+With regular reprocessing of datasets, the volume of data on disk will grow rapidly. It is neither necessary nor feasible to preserve all processed datasets in perpetuity. The following gives the required timescales for retaining processed test datasets:
+
+ * LARGE: A minimum of two datasets should always be preserved as well as two sets of corresponding master calibraions to be used for subsequent processing campaigns. The reason is to be able to compare the results of each subsequent processing campaign. One of the two may be deleted prior to processing the next one if space is needed. 
+ * MEDIUM: A minimum of 12 months.
+ * SMALL: A maximum of 1 month.
```
Comment (yalsayyad) [inline index.rst]: Whats does a "maximum" timescale to preserve data mean in this scenario?

Comment (yalsayyad) [reply index.rst]: Also, I don't really know what preserving a SMALL dataset (e.g. validation_data_hsc and testdata_ci_hsc. I know @SimonKrughoff  maintains raws for these repos in `/project/shared/` (Thank you, I use them all the time,) but I don't know of any preserved outputs (mainly because I wouldn't need them).

Comment (gpdf) [reply index.rst]: Preserved outputs are very useful for people testing downstream components, who, not being pipelines developers, don't find it as trivial to regenerate them at need.  See the infamous RFC-243.

I think a "maximum" time is expressed for the "small" dataset because the idea is that it's just a running snapshot of the nature of the output data from the pipelines.

Comment (yalsayyad) [reply index.rst]: @gpdf I'm being really dense and need an ELI5. Remember I'm coming from regular large/medium reprocessing on lsst-dev land.

I get that a *minimum* means that you must save it for > 12 mo. So 12 mo pass and now you can delete it whenever you get around to it and need space.    A *maximum* would mean that you must save it for < 1mo. How does that work? If a requirement was < 1mo, why not delete it after 1 second.  How do you enforce that someone deletes it after 29 days? 

"Preserved outputs are very useful for people" is self-evident. Remember I'm on spend all the $ side. :)  No new info there.    *What/where* are the *small* preserved *outputs*?  Are they in the cloud somewhere? Do they not exist yet?

Comment (gpdf) [reply index.rst]: All good points.  

My saying that I assessed preserved outputs as useful was, I thought, just expressing a view from a different corner of DM to put along side your statement that you don't need preserved outputs from small runs.

I was trying to be generous in interpreting the "maximum" language and treating it like a design point, expressing expected need rather than a requirement on preservation.  To me it reads like a reassurance like "don't worry, this won't become a big accumulation; no way do are we going to need this data for more than a month".  

I'd be happy to suggest something more actionable, e.g., "Datasets in this category should be managed so that there is always at least one available and so that the likelihood of a dataset being deleted while in use is mitigated.  The output from each successive run in this category should be preserved at least until 72 hours after the output of the next run is available."

As to where they are to be saved, I would have thought, at least for now, on GPFS at NCSA.  And as for the "what?" I think this, ultimately, would be a representative selection of both pipeline-output and SDM-standardized data products from both AP and DRP.  I thought this was already relatively well covered in the DMTN apart from the issue of retaining both directly pipeline output and standardized output.

Comment (leannep) [reply index.rst]: @gpdf's reading of "maximum time" is what I intended  -- "no way do are we going to need this data for more than a month"  but it seems to be confusing so I'll make it clearer

Comment (leannep) [reply index.rst]: As discussed off line, Gregory's suggested text has been incorporated.
